webpack.config.js

```diff
@@ -10,7 +10,7 @@ module.exports = {
     },
     output: {
         path: path.resolve(__dirname, 'dist'),
-        clean: true,
+        clean: false,
```

Comment:
Why aren't we cleaning out the dist folder when it builds?

Reply:
Fixed